app/uk/gov/hmrc/tariffclassificationfrontend/views/components/type_description.scala.html

```diff
@@ -0,0 +1,30 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{Application, LiabilityStatus}
+
+@(application: Application)
+
+<div>
+    @if(application.isBTI) {
+        @application.getType
+    } else {
+        Liability</br>
```

Comment:
You can use application.getType for liabilities as well now

Reply:
will do